ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/tunnel-interfaces.j2

```diff
@@ -42,6 +42,9 @@
 | {{ tunnel_interface.name }} | {{ row_vrf }} | {{ row_ip_addr }} | {{ row_tcp_mss }} | {{ row_tcp_mss_direction }} | {{ row_acl_in }} | {{ row_acl_out }} |
 {%         endfor %}
 {%     endif %}
+{# IP NAT Documentation #}
+{%     set ip_nat_interfaces = tunnel_interfaces | arista.avd.default({}) %}
```

Comment:
```suggestion
{%     set ip_nat_interfaces = tunnel_interfaces | arista.avd.default([]) %}
```

Reply:
As I will remove NAT support on tunnel interfaces this code section will be completely removed.